src/List/web/ListItem.js

```diff
@@ -3,8 +3,7 @@ import styled from 'styled-components';
 
 const ListItem = styled.li`
   color: ${(props) => props.disabled ? props.theme.color.greyLight : ''};
-  padding: ${(props) => props.theme.px(2)};
-  cursor: ${(props) => props.onClick ? 'pointer' : ''};
+  cursor: ${(props) => props.clickable ? 'pointer' : ''};
```

Comment:
There was a change in the newer version, `props.clickable` was changed to `props.onClick`